src/interpreter_funcs/voxel_boolean_intersection.rs

```diff
@@ -0,0 +1,261 @@
+                In some cases not growing the volume at all may result in \
+                a non manifold voxelized mesh.",
+                refinement: ParamRefinement::Uint(UintParamRefinement {
+                    default_value: Some(0),
+                    min_value: None,
+                    max_value: None,
+                }),
+                optional: false,
+            },
+            ParamInfo {
+                name: "Fill Closed Volumes",
+                description: "Treats the insides of watertight mesh geometries as volumes.\n\
+                \n\
+                If this option is off, the resulting voxelized mesh geometries will have two \
+                separate mesh shells: one for outer surface, the other for inner surface of \
+                hollow watertight mesh.",
+                refinement: ParamRefinement::Boolean(BooleanParamRefinement {
+                    default_value: true,
+                }),
+                optional: false,
+            },
+            ParamInfo {
+                name: "Prevent Unsafe Settings",
+                description: "Stop computation and throw error if the calculation may be too slow.",
+                refinement: ParamRefinement::Boolean(BooleanParamRefinement {
+                    default_value: true,
+                }),
+                optional: false,
+            },
+            ParamInfo {
+                name: "Bounding Box Analysis",
+                description: "Reports basic and quick analytic information on the created mesh.",
+                refinement: ParamRefinement::Boolean(BooleanParamRefinement {
+                    default_value: true,
+                }),
+                optional: false,
+            },
+            ParamInfo {
+                name: "Detailed Mesh Analysis",
+                description: "Reports detailed analytic information on the created mesh.\n\
+                              The analysis may be slow, therefore it is by default off.",
+                refinement: ParamRefinement::Boolean(BooleanParamRefinement {
+                    default_value: false,
+                }),
+                optional: false,
+            },
+        ]
+    }
+
+    fn return_ty(&self) -> Ty {
+        Ty::Mesh
+    }
+
+    fn call(
+        &mut self,
+        args: &[Value],
+        log: &mut dyn FnMut(LogMessage),
+    ) -> Result<Value, FuncError> {
+        let mesh1 = args[0].unwrap_mesh();
+        let mesh2 = args[1].unwrap_mesh();
+        let voxel_dimensions = Vector3::from(args[1].unwrap_float3());
+        let growth_u32 = args[3].unwrap_uint();
+        let growth_f32 = growth_u32 as f32;
+        let fill = args[4].unwrap_boolean();
+        let error_if_large = args[5].unwrap_boolean();
+        let analyze_bbox = args[6].unwrap_boolean();
+        let analyze_mesh = args[7].unwrap_boolean();
+
+        if voxel_dimensions
+            .iter()
+            .any(|dimension| approx::relative_eq!(*dimension, 0.0))
```

Comment:
Same as in "Difference", let's include the possibility of negative numbers in the check.